src/dal/Jalasoft.TeamUp.Projects.DAL/ProjectsMongoDbRepository.cs

```diff
@@ -10,26 +10,33 @@
     public class ProjectsMongoDbRepository : IProjectsMongoDbRepository
     {
         private static MongoClient client;
+        private static IMongoDatabase database;
+        private static IMongoCollection<Project> collection;
 
         public ProjectsMongoDbRepository()
         {
-            string stringConnection = ConfigurationManager.ConnectionStrings["MongoSessionServices"].ConnectionString;
+            string stringConnection = Environment.GetEnvironmentVariable("MongoSessionServices", EnvironmentVariableTarget.Process);
             client = new MongoClient(stringConnection);
+            database = client.GetDatabase("Projects");
+            collection = database.GetCollection<Project>("Projects");
         }
 
         public Project Add(Project project)
         {
-            throw new NotImplementedException();
+            collection.InsertOne(project);
+            return project;
         }
 
         public IEnumerable<Project> GetAll()
         {
-            throw new NotImplementedException();
+            List<Project> projects = collection.Find(x => true).ToList();
+            return projects;
         }
 
         public Project GetById(Guid id)
         {
-            throw new NotImplementedException();
+            Project project = collection.Find(x => x.Id == id).Single();
```

Comment:
What happens when there isn't a project with that specific GUI?

Reply:
GetById method returns null so API Controller return a NotFoundObjectResult